mozdownload/scraper.py

```diff
@@ -863,7 +862,6 @@ def platform_regex(self):
         PLATFORM_FRAGMENTS = {'linux': 'linux',
                               'linux64': 'linux64',
                               'mac': 'macosx',
-                              'mac64': 'macosx64',
```

Comment:
Not sure why you are getting this portion removed. We only have macosx64 folders for tinderbox builds like `mozilla-central-macosx64.*/`. It would be better to say that `mac` uses those too, if not done yet.